client/signup/steps/user/index.jsx

```diff
@@ -166,8 +206,8 @@ export class UserStep extends Component {
 		if ( positionInFlow === 0 && flowName === 'onboarding' ) {
 			subHeaderText = translate( 'First, create your WordPress.com account.' );
 
-			if ( this.props.isReskinned ) {
-				const loginUrl = this.getLoginLink();
+			if ( isReskinned ) {
+				const loginUrl = getLoginLink( props );
```

Comment:
Can't we call `login()` already here and remove `getLoginLink` function?

Reply:
We most certainly can. It's only used here.